frontend/src/components/SummaryView/index.jsx

```diff
@@ -45,18 +54,21 @@ function SummaryView({
 
     const avgData = filterData.map(y => ({group: y.name, subgroups: checkedColorCategoryLevels.map(x => ({name: x.value, avg: _.meanBy(y.value.filter(z => x.value == z[groupAttr]).map(v => Number((v[yAxisAttr]))))}))}));
 
+    // get largest avg value for y axis height
+    var maxAvg = 0;
+    avgData.forEach(d => d.subgroups.forEach(s => { if (s.avg > maxAvg) { maxAvg = s.avg}}));
```

Comment:
Can you make sure the average value is shown to 2 decimal places in the tooltip?